core/adapters/eth_tx_test.go

```diff
@@ -36,8 +35,7 @@ func TestEthTxAdapter_Perform_BPTXM(t *testing.T) {
 			DataPrefix:       hexutil.MustDecode("0x0000000000000000000000000000000000000000000000000000000000000001"),
 			ABIEncoding:      []string{"bytes32", "uint256", "bool", "bytes"},
 		}
-		jobRunID := uuid.NewV4()
```

Comment:
Needs to be the real job run ID as we now use that to look it up when performing an EthTx task